### DISCUSSION TITLE
Why is there horizontal and vertical spacing tokens?

### DISCUSSION THREAD

**Discussion by tombohub** [microsoft/fluentui#30449] [Q&A] [answered]
in token docs: https://react.fluentui.dev/?path=/docs/theme-spacing--page

there are same values for spacing horizontal and same values for spacing vertical. What's the reason behind and not have only one spacing?

**Comment by khmakoto** [comment, marked as answer]
The reason is that different `themes` could have a different ramp of values for horizontal and vertical spacing. We do not have this in our themes but realize that is something that could be needed in other themes/later on, so our token infrastructure allows for it.
